tidb.go

```diff
@@ -88,7 +89,7 @@ var (
 	}
 	stores = make(map[string]kv.Driver)
 	// Debug is the switch for pprof. If Debug is true, tidb will start with pprof on.
-	Debug = true
+	Debug = os.Getenv("TIDB_DEBUG") == "1"
```

Comment:
Even though I agree debug should be true by default, but I don't think start listening on a port is desired behavior of users. 

Maybe we should use `TIDB_PPROF=1`?